Intersections_3/include/CGAL/internal/Intersections_3/Triangle_3_Line_3_intersection.h

```diff
@@ -394,12 +394,9 @@ intersection(const typename K::Triangle_3 &t,
             else
               return intersection_return<typename K::Intersect_3, typename K::Line_3, typename K::Triangle_3>();
 
-          case COPLANAR: // pqa or pqb or pqc are collinear
+          default: // COPLANAR: // pqa or pqb or pqc are collinear
```

Comment:
Indentation.

Reply:
Sorry. That one is OK. I had to extend the scope of the Github diff to see the neighboring code.